Real name for arm926t is arm926ej-s. arm926t is the old name still used in
Linux & uClibc.

Signed-off-by: Julien Boibessot <julien.boibessot@armadeus.com>

Index: buildroot/target/Config.in.arch
===================================================================
--- buildroot.orig/target/Config.in.arch	2012-02-21 16:43:25.000000000 +0100
+++ buildroot/target/Config.in.arch	2012-02-21 16:45:32.000000000 +0100
@@ -68,7 +68,7 @@
 config BR2_arm922t
 	bool "arm922t"
 config BR2_arm926t
-	bool "arm926t"
+	bool "arm926ej-s (arm926t)"
 config BR2_arm10t
 	bool "arm10t"
 config BR2_arm1136jf_s
